viam_cartographer_test.go

```diff
@@ -113,24 +113,6 @@ func TestNew(t *testing.T) {
 		test.That(t, svc.Close(context.Background()), test.ShouldBeNil)
 	})
 
-	t.Run("Successful creation of cartographer slam service with good lidar with movement"+
-		"sensor that does not support IMU nor odometer", func(t *testing.T) {
```

Comment:
Removed this, because we're now failing if a movement sensor does not support at least an IMU or odometer.